.travis.yml

```diff
@@ -27,6 +31,7 @@ before_script:
   - sleep 4
   - toxiproxy-cli create redis -l localhost:26379 -u localhost:6379
   - sleep 1
+  - mix compile --warnings-as-errors
```

Comment:
could you move this to `script:`? I prefer the errors are raised from script block rather than before_script

Reply:
Sure. Just rebased and pushed a commit to move this. 👍